tools/cli/commands/build.js

```diff
@@ -1065,7 +1074,13 @@ async function buildProject( t ) {
 		runversion: projectRunVersionNumber,
 	};
 	await t.ctx.mirrorMutex( async () => {
-		// prettier-ignore
-		await fs.appendFile( `${ t.argv.forMirrors }/mirrors.txt`, `${ gitSlug }\n`, { encoding: 'utf8' } );
+		// Priority repos are pushed to the mirror repos first so we can deploy them sooner.
+		const priorityRepos = [ 'Automattic/jetpack-production', 'Automattic/jetpack-mu-wpcom-plugin' ];
```

Comment:
If we're going to store these in an object, I'd put it at the top of the file and make it a `Set` instead of an array instead of having to initialize it every time code reaches this point.

Reply:
4907ed2